### DISCUSSION TITLE
wanted some help regarding a problem im facing while hosting my webpage through github pages

### DISCUSSION THREAD

**Discussion by Hritik2108** [community/community#63642] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

i have just created a project for my college and created a github repo and generated the link for the website but the problem is I am able to view the website perfectly just the way i made it but when I'm sharing the link to others the components of the webpage are getting displaced. please help me out on this here's the link https://hritik2108.github.io/Library.github.io/ 
![Library and 5 more pages - Personal - Microsoft​ Edge 13-08-2023 23_23_48](https://github.com/community/community/assets/108113370/b2f3305f-f2ae-4f44-baf1-ebef7b1074e0)
i am seeing like this but its getting destroyed in the shared link

**Comment by MayaAtwan** [comment]
hey,
it could be tied to responsive design aspects, Responsive design ensures the site looks great on various screens. Could you test it on different devices or browsers to help pinpoint the problem?

**Reply by Hritik2108** [reply]
in my device it is working okay with different browser bt in other device is not working the same on the same browser which I used...u can check the link and the image....kindly check the link and reply me is it getting displaced in your device as well

**Reply by MayaAtwan** [reply]
I checked out the link and noticed the displacement issue.are you using responsive images on your website? They can sometimes affect the layout on different devices.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Hritik2108 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
